### PR TITLE
Send Git info when not on Travis CI

### PR DESCRIPTION
**Pull request by rimmington**
Only on non-Travis builds because Coveralls seems to do some magic for that service that I don't want to override.

### CODE REVIEW

File: hpc-coveralls.cabal [hunk 1]
```diff
@@ -60,6 +60,7 @@ library
     cmdargs >= 0.10,
     curl >= 1.3.8,
     hpc >= 0.6,
+    process >= 1.1.0.1,
```
Comment (guillaume-nargeot) [inline hpc-coveralls.cabal]: Is there any reason for this particular version? (there seems to be newer ones)

**Comment by guillaume-nargeot** [comment]
Thank you very much for the contribution!
Installing with the package produced by `cabal sdist` is failing because the `Trace.Hpc.Coveralls.GitInfo` module has to be added to the `library.other-modules` section in `hpc-coveralls.cabal`.
I will make this change after merging.

File: hpc-coveralls.cabal (hunk 1 above)
Comment (rimmington) [inline hpc-coveralls.cabal]: It's just the first version I could confirm to have `readProcess`, and so seemed to be the logical lower bound.

Comment (guillaume-nargeot) [inline hpc-coveralls.cabal]: Ok understood, thanks!

**Pull request merged**